exo-go/src/docker_compose/index.go

```diff
@@ -0,0 +1,43 @@
+package dockerCompose
+
+import (
+	"bufio"
+	"bytes"
+	"fmt"
+	"os/exec"
+)
+
+// RunAllImages runs all the docker images
+func RunAllImages(env []string, cwd string, write func(string)) (*exec.Cmd, *bytes.Buffer, error) {
```

Comment:
Okay, good find! We definitely want to use Docker Compose version 3. That might be the killer argument here.